src/Tribe/iCal.php

```diff
@@ -75,12 +75,15 @@ public function single_event_links() {
 		if ( is_single() && post_password_required() ) {
 			return;
 		}
-		echo '<div class="tribe-events-cal-links">';
-		echo '<a class="tribe-events-gcal tribe-events-button" href="' . Tribe__Events__Main::instance()->esc_gcal_url( tribe_get_gcal_link() ) . '" title="' . esc_attr__( 'Add to Google Calendar', 'the-events-calendar' ) . '">+ ' . esc_html__( 'Google Calendar', 'the-events-calendar' ) . '</a>';
-		echo '<a class="tribe-events-ical tribe-events-button" href="' . esc_url( tribe_get_single_ical_link() ) . '" title="' . esc_attr__( 'Download .ics file', 'the-events-calendar' ) . '" >+ ' . esc_html__( 'iCal Export', 'the-events-calendar' ) . '</a>';
-		echo '</div><!-- .tribe-events-cal-links -->';
+		$calendar_links = '<div class="tribe-events-cal-links">';
+		$calendar_links .= '<a class="tribe-events-gcal tribe-events-button" href="' . Tribe__Events__Main::instance()->esc_gcal_url( tribe_get_gcal_link() ) . '" title="' . esc_attr__( 'Add to Google Calendar', 'the-events-calendar' ) . '">+ ' . esc_html__( 'Google Calendar', 'the-events-calendar' ) . '</a>';
+		$calendar_links .= '<a class="tribe-events-ical tribe-events-button" href="' . esc_url( tribe_get_single_ical_link() ) . '" title="' . esc_attr__( 'Download .ics file', 'the-events-calendar' ) . '" >+ ' . esc_html__( 'iCal Export', 'the-events-calendar' ) . '</a>';
+		$calendar_links .= '</div><!-- .tribe-events-cal-links -->';
+
+		echo apply_filters( 'tribe_events_single_links', esc_html( $calendar_links ) );
```

Comment:
The text contains HTML, so we probably want to drop the use of `esc_html()` here :-)

Reply:
@barryhughes would you think something like `wp_kses_post()` is unnecessary and expensive, or would that be cool to use instead?

Reply:
Not completely closed to the idea, but I don't feel it's necessary/it's not something we generally do in this sort of situation.

Reply:
Right on.